stackscli.yml

```diff
@@ -4,10 +4,10 @@ pipeline:
       - name: build
         path: build/azDevOps/azure/templates/air-infrastructure-data.yml
       - name: variable
-        path: build/azDevOps/azure/air-infrastructure-data-vars.yml
+        path: build/common-vars.yml
```

Comment:
the common-vars-template.yml is the only file that needs to be replaced by by stacks-cli